.github/workflows/deploy.yml

```diff
@@ -31,3 +31,20 @@ jobs:
           PIPEDRIVE_BUMP_TOKEN:  ${{ secrets.PIPEDRIVE_BUMP_TOKEN }}
           GITLAB_BUMP_TOKEN: ${{ secrets.GITLAB_BUMP_TOKEN }}
           REALWORLD_BUMP_TOKEN: ${{ secrets.REALWORLD_BUMP_TOKEN }}
+          ASI_BUMP_TOKEN: ${{ secrets.ASI_BUMP_TOKEN }}
+          AXONAUT_BUMP_TOKEN: ${{ secrets.AXONAUT_BUMP_TOKEN }}
+          FAIRJUNGLE_BUMP_TOKEN: ${{ secrets.FAIRJUNGLE_BUMP_TOKEN }}
+          GALISE_BUMP_TOKEN: ${{ secrets.GALISE_BUMP_TOKEN }}
+          GITGUARDIAN_BUMP_TOKEN: ${{ secrets.GITGUARDIAN_BUMP_TOKEN }}
+          HIBOUTIK_BUMP_TOKEN: ${{ secrets.HIBOUTIK_BUMP_TOKEN }}
+          INPI_API_RNCS_BUMP_TOKEN: ${{ secrets.INPI_API_RNCS_BUMP_TOKEN }}
+          LUMAPPS_BUMP_TOKEN: ${{ secrets.LUMAPPS_BUMP_TOKEN }}
+          MAPOTEMPO_BUMP_TOKEN: ${{ secrets.MAPOTEMPO_BUMP_TOKEN }}
+          NEXTAUTH_BUMP_TOKEN: ${{ secrets.NEXTAUTH_BUMP_TOKEN }}
+          PAPPERS_BUMP_TOKEN: ${{ secrets.PAPPERS_BUMP_TOKEN }}
+          PAYGREEN_BUMP_TOKEN: ${{ secrets.PAYGREEN_BUMP_TOKEN }}
+          REMOVE_BG_BUMP_TOKEN: ${{ secrets.REMOVE_BG_BUMP_TOKEN }}
+          RESTAURANTS_API_BUMP_TOKEN: ${{ secrets.RESTAURANTS_API_BUMP_TOKEN }}
+          SKALEET_BUMP_TOKEN: ${{ secrets.SKALEET_BUMP_TOKEN }}
+          SLACK_API_BUMP_TOKEN: ${{ secrets.SLACK_API_BUMP_TOKEN }}
+          TALEEZ_BUMP_TOKEN: ${{ secrets.TALEEZ_BUMP_TOKEN }}
```

Comment:
Nice thanks a lot for this! Did you add the secret variables on the repo settings taken from  the token values of each APIs on Bump.sh?